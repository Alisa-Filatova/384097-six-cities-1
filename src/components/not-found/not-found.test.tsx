import * as React from 'react';
import * as renderer from 'react-test-renderer';
import NotFound from './not-found';

describe(`NotFound`, () => {
  it(`renders correctly`, () => {
    const page = renderer.create(<NotFound />).toJSON();
    expect(page).toMatchSnapshot();
  });
});
