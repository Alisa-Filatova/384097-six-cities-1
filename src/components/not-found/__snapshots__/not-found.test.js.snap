// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders correctly 1`] = `
<div
  className="cities__places-wrapper"
  style={
    Object {
      "height": "100vh",
    }
  }
>
  <div
    className="cities__places-container cities__places-container--empty container"
  >
    <section
      className="cities__no-places"
    >
      <div
        className="cities__status-wrapper tabs__content"
      >
        <b
          className="cities__status"
        >
          Page not found (404)
        </b>
        <p
          className="cities__status-description"
        >
          Oops! The page you were looking for does not exist
        </p>
      </div>
    </section>
    <div
      className="cities__right-section"
    />
  </div>
</div>
`;
